src/dso_api/dynamic_api/views/wfs.py

```diff
@@ -97,7 +97,8 @@ def get_environments(self, ds: Dataset, base: str):
                 "name": "production",
                 "api_url": base + reverse("dynamic_api:wfs", kwargs={"dataset_name": ds.name}),
                 "specification_url": base
-                + reverse("dynamic_api:wfs", kwargs={"dataset_name": ds.name}),
+                + reverse("dynamic_api:wfs", kwargs={"dataset_name": ds.name})
+                + "?SERVICE=WFS&REQUEST=GetCapabilities",
```

Comment:
Ik weet niet of dit zo handig is. In QGis (en veel anderen) moet je juist de URL zonder deze parameters gebruiken.
Als gebruikers deze URL gaan kopieren fixeren ze juist die parameter en krijgen we veel vragen waarom de WFS niet werkt.

Reply:
Hmmm... dat weet ik niet hoor. Dit was op mijn aanraden. Het is het equivalent van de OpenAPI Spec voor OGC API's ... en volgens mij kan je die ook in QGIS plakken en pakt hij dat ook op...

Reply:
Maar dit is de specificatie url. De hoofd url is `api_url`, dat is wat mensen in QGis inladen toch?.

Reply:
Nee, dat kan je helaas niet in QGis plakken. Ik heb daar support op moeten leveren, en daarom staat het ook in de docs nadrukkelijk als "tip" vermeld dat je niet die parameters er zelf achter moet zetten: https://api.data.amsterdam.nl/v1/docs/generic/wfs.html

Reply:
Denk dat ze dat gefixed hebben, bij mij werkt het (QGis 3.18.3-Zürich). De query params worden van de url geknipt in het invoerveld. 
Maar zowel de pagina op 'api_url' als de docs_url verwijzen naar de juiste invoer voor QGis. Het zou een beetje raar om niet naar de specs te verwijzen bij een ding dat "specification_url" heet. Als je bang bent dat het verwarrend blijft voor mensen met een oude versie, dan kunnen we  "specification_url" misschien beter weglaten bij wfs?